interactive_ai/services/resource/app/communication/rest_endpoints/media_endpoints.py

```diff
@@ -112,13 +115,19 @@ def post_image(
         upload_size=int(request.headers["content-length"]),
         exception_type=NotEnoughSpaceException,
     )
-    return MediaRESTController.upload_media(
+    content = MediaRESTController.upload_media(
         user_id=user_id,
         dataset_storage_identifier=dataset_storage_identifier,
         label_info=upload_info,
         media_type=MediaType.IMAGE,
         file_from_request=file,
     )
+    status_code = (
+        status.HTTP_202_ACCEPTED
+        if FeatureFlagProvider.is_enabled(FeatureFlag.FEATURE_FLAG_ASYNCHRONOUS_MEDIA_PREPROCESSING)
+        else status.HTTP_200_OK
+    )
```

Comment:
For reference: REST API docs will be updated in [ITEP-67407](https://jira.devtools.intel.com/browse/ITEP-67407)